woocommerce/woocommerce
# [CYS] Stop relying on snapshot for footer and header tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Run:
- `pnpm run --filter="@woocommerce/plugin-woocommerce" test:e2e-pw plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js`
- `pnpm run --filter="@woocommerce/plugin-woocommerce" test:e2e-pw plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/header.spec.js`

Make sure all tests pass.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
[CYS] Update footer and header test to not use snapshots.
#### Comment 

</details>
